Cargo.toml

```diff
@@ -22,43 +22,42 @@ exclude = [
 
 [dependencies]
 backoff         = { version = "0.4.0", optional = true }
-base64          = "^0.13"
-basic-cookies   = { version = "^0.1", optional = true }
+base64          = "0.22.1"
+basic-cookies   = { version = "0.1", optional = true }
 bytes           = "1"
-chrono          = { version = "^0.4", features = ["serde"] }
+chrono          = { version = "0.4", features = ["serde"] }
 clap            = "^2.33"
 cryptoki        = { version = "^0.3", optional = true }
 cryptoki-sys    = { version = "=0.1.4", optional = true } # pin cryptoki-sys because of compilation issues on various systems
-fern            = { version = "^0.5", features = ["syslog-4"] }
-futures         = "^0.3"
+fern            = { version = "0.6.2", features = ["syslog-6"] }
 futures-util    = "^0.3"
```

Comment:
Given the removal of the `^` prefix on most other dependencies, why do some retain a leading `^`?